production/user_guide.md

```diff
@@ -125,142 +125,158 @@ You can repeat this process with the `/_version` and `/api/v1/metadata` endpoint
 
 The `Response Body` contains the requested claims data in NDJSON format. Click the `Download` button that appears in the lower right corner of the response section. Note that a large file may take a while to download.
 
-<img class="ug-img" src="/assets/img/swagger_walkthrough_12.svg" alt="" />
+<img class="ug-img" src="/assets/img/swagger_walkthrough_12.svg" alt="">
 
 If you have requested data related to more than one Resource Type, the files related to each Resource Type will appear separately.
 
 You will have twenty minutes before your token expires, and you will need to get another from `/auth/token` if it expires before you are finished interacting with the API.
 
-Once you’ve downloaded the file, you’ll want to know what to do with the data. We’ve provided a [guide to working with BCDA data](/data-guide/){:target="_self"} to help you, including a crosswalk between CCLF fields and the corresponding sections of the NDJSON files.
+Once you’ve downloaded the file, you’ll want to know what to do with the data. We’ve provided a [guide to working with BCDA data](https://bcda.cms.gov/data-guide/){:target=”_blank”} to help you, including a crosswalk between CCLF fields and the corresponding sections of the NDJSON files.
 
+#### Filtering your requests using _since
+The `_since` parameter lets you filter your bulk data requests by the date when the data was updated. This means that instead of receiving all your historical data each time you request data from the API, you will instead be able to enter the date since you last requested data and receive only data updated between your `_since` input date and the present.
 
-### Filtering your requests using `_since`
+The `_since` parameter can be used with requests for all resource types. 
 
-The `_since` parameter lets you filter your bulk data requests by the date when the data was updated. This means that instead of receiving all your historical data each time you request data from the API, you will instead be able to enter the date since you last requested data and receive only data updated between your `_since` input date and the present. The `since` parameter can be used with requests for all resource types.
+**Note: The use of `_since` differs significantly between the `/Patient` and `/Group` endpoints:**
 
-Before using `_since` for the first time, pull your historical data. Using the `_since` parameter subsequently comprises three steps:
+* Using `_since` with the `Patient` endpoint will return data for all beneficiaries filtered by date.
+* Using `_since` with the `/Group` endpoint will return data for only some beneficiaries filtered by date. Beneficiaries who were attributed to your ACO more recently (after your latest attribution date) are considered ***new beneficiaries***. You will receive all 7 years of historical data for new beneficiaries with the use of `_since`. Data for older beneficiaries (called ***existing beneficiaries***) will be filtered by the `_since` date.
```

Comment:
> You will receive all 7 years of historical data for new beneficiaries with the use of `_since`.

Small stylistic thing: Would personally change "with the use of" to "when using" -- but that is your call!